packages/nouns-webapp/src/components/AuctionActivity/index.tsx

```diff
@@ -28,80 +28,37 @@ const openEtherscanBidHistory = () => {
 
 interface AuctionActivityProps {
   auction: Auction;
-  isFirstAuction: boolean;
-  isLastAuction: boolean;
-  onPrevAuctionClick: () => void;
-  onNextAuctionClick: () => void;
   displayGraphDepComps: boolean;
 }
 
 const AuctionActivity: React.FC<AuctionActivityProps> = (props: AuctionActivityProps) => {
-  const {
-    auction,
-    isFirstAuction,
-    isLastAuction,
-    onPrevAuctionClick,
-    onNextAuctionClick,
-    displayGraphDepComps,
-  } = props;
+  const { auction, displayGraphDepComps } = props;
 
   const isCool = useAppSelector(state => state.application.isCoolBackground);
-
-  const [auctionEnded, setAuctionEnded] = useState(false);
-  const [auctionTimer, setAuctionTimer] = useState(false);
+  const lastNounId = useAppSelector(state => state.onDisplayAuction.lastAuctionNounId);
+  const isLastAuction = auction.nounId.toNumber() === lastNounId;
+  const isVrgda = isVrgdaNoun(auction.nounId.toNumber());
```

Comment:
Using `isVrgdaNoun` to determine whether past auction was VRGDA if not. Need to do it to e.g. hide bid history when non-applicable.